pkg/noderesourcetopology/cache/overallocating.go

```diff
@@ -49,9 +49,10 @@ type Overallocating struct {
 	nodeTainted      counter
 	lister           listerv1alpha1.NodeResourceTopologyLister
 	indexer          NodeIndexer
+	statusDirectory  string
```

Comment:
Pods are replaceable units, what would happen if the plugin pod gets crashed, restarted, etc?
Do we keep a copy of the file somewhere on the host?

Reply:
we lose this data, and it's fine. It's ephemeral reporting data anyway. But it's also half-baked borderline hack, so let's postpone this.